packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe core SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Modified Services.ts to have commented properties end with commas [#1049](https://github.com/zowe/zowe-cli/issues/1049)
```

Comment:
- Can we add a modifier to Services.ts to explain what it is?: "Modified the property/profile/biscuit/whatever Services.ts to ..."
- Please add a period after "commas."